scripts/lc-builds/toss4_icpc.sh

```diff
@@ -58,6 +58,6 @@ echo
 echo "  Please note that you may need to add some intel openmp libraries to your"
 echo "  LD_LIBRARY_PATH to run with openmp."
 echo
-echo "    LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:/usr/tce/packages/intel-classic-tce/intel-classic-2021.6.0/compiler/lib/intel64_lin"
+echo "    LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:/usr/tce/packages/intel/intel-2022.3/compiler/lib/intel64_lin"
```

Comment:
It's probably good to mimic the compiler path in the CMake options above, because this specific line may not work if a different compiler version is chosen for the build.

Reply:
Whoops, I had that change but didn't propagate it correctly.